packages/react/src/bags/hooks/__tests__/useBag.test.tsx

```diff
@@ -629,7 +654,6 @@ describe('useBag', () => {
             authCode: undefined,
             customAttributes: '',
             merchantId: mockMerchantId,
-            productAggregatorId: undefined,
```

Comment:
You can keep this line to ensure that the productAggregatorId when we do not specify it keeps being undefined.